### ISSUE TITLE
Question about unknown/unknown Tag in ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator Docker Image

### ISSUE THREAD

**Issue by wzy531a** [open-telemetry/opentelemetry-operator#3134]
### Describe the issue you're reporting

Hi,

I noticed that the Docker image [ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator/pkgs/container/opentelemetry-operator%2Fopentelemetry-operator) contains a tag labeled unknown/unknown. Can someone clarify what this tag represents and what its purpose is?

Thank you!

**Comment by jaronoff97** [comment]
@wzy531a can you link that image? I looked through GHCR and couldn't find it.

**Comment by wzy531a** [comment]
> @wzy531a can you link that image? I looked through GHCR and couldn't find it.

Yes, when I tried to pull the unknown image, Docker indicated that the operating system is not supported. For now, I tried the following:
`[root@k8s-master ~]# docker pull ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:main@sha256:bdf756140620a2e71239185c1fc136291d2ea78c65c36ed861cc5c87f65129fb

ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator@sha256:bdf756140620a2e71239185c1fc136291d2ea78c65c36ed861cc5c87f65129fb: Pulling from open-telemetry/opentelemetry-operator/opentelemetry-operator
e00d8f0bad61: Pulling fs layer
operating system is not supported`

**Comment by wzy531a** [comment]
> what OS are you using?

I am using Linux-x64. The 'unknown' tag is provided for systems where the specific OS is not identified. If the system is not Linux, you can try using the 'unknown' tag image.

**Comment by jaronoff97** [comment]
Im unclear on what the issue you're experiencing is now. Can you share more information, including logs and OS.

**Comment by wzy531a** [comment]
> Im unclear on what the issue you're experiencing is now. Can you share more information, including logs and OS.

I want to know what the meaning of unknown OS/Arch is?
For example, Linux/amd64 corresponds to a Linux system running on a 64 bit AMD architecture, while Linux/arm64 corresponds to a Linux system running on a 64 bit ARM architecture. So, what is unknown

**Comment by pavolloffay** [comment]
![Screenshot of opentelemetry-operator_opentelemetry-operator versions · open-telemetry](https://github.com/user-attachments/assets/f6c7321e-5cbb-43b3-aebb-bcd8e6974c02)


Adding a screenshot that shows the label.

**Comment by pavolloffay** [comment]
Here it is explained why unknown/unknown is shown https://github.com/orgs/community/discussions/45969

**Comment by pavolloffay** [comment]
@wzy531a would you like to submit a patch for it?

**Comment by gautamkrishnar** [comment]
@pavolloffay i had added a PR to fix this issue. I am not sure if a fix for this is still required. 

Thanks in advance.

**Comment by oespallargas** [comment]
Revisiting this issue, looks like only AMD64 images available now?
